sound/soc/sof/topology.c

```diff
@@ -2037,12 +2039,12 @@ static int sof_process_load(struct snd_soc_component *scomp, int index,
 	struct snd_sof_dev *sdev = snd_soc_component_get_drvdata(scomp);
 	struct snd_soc_dapm_widget *widget = swidget->widget;
 	struct snd_soc_tplg_private *private = &tw->priv;
-	struct sof_ipc_comp_process *process = NULL;
+	struct sof_ipc_comp_process *process;
```

Comment:
separate patch @lyakh. I know its painful but if I dont say it, someone will point it out in the upstream reviews.